### PR TITLE
Create guiding-principles

### PR DESCRIPTION
**Pull request by WITMiddle**
## Overview

As part of contribution to RA Wiki, adding Security Standards. This PR is for "Guiding Principles" section under Security.

### Features

- add "Guiding Principles" to Security

### CODE REVIEW

File: security/guiding-principles.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Guiding Principles
+
+- Be upfront, clear and transparent about how information is collected, used and disclosed.
+- Follow Canada’s Anti-Spam Legislation (CASL).
```
Comment (crimeminister) [inline security/guiding-principles.md]: Couldn't hurt to link to an information site for those curious about this legislation, e.g. http://fightspam.gc.ca/.

File: security/guiding-principles.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- Solicit from customers only data points that are absolutely required. Avoid (or minimize) the amount of “optional” information collected.
+- Adhere to principle of least privilege - only grant access to those who require access for a legitimate business purpose. Access should be further restricted to least privilege required to accomplish a business task. (i.e Write/Modify/Read).
+- Collection and storage of information for a web property requires a Privacy Impact Assessment (PIA), Security Architecture Engagement request (SAER) and Security Analysis Engagement (SAE). These are internal processes that need to be coordinated by a TELUS prime if you are a digital vendor.
```
Comment (crimeminister) [inline security/guiding-principles.md]: Please add x-links to the location where it is described how to initiate these business processes.

Comment (WITMiddle) [reply security/guiding-principles.md]: I actually decided to remove this section and associated PR. Thank you for taking the time to review this, I will integrate the content and your feedback into other sections.

Comment (crimeminister) [reply security/guiding-principles.md]: Thanks @WITMiddle. I'm looking forward to having all this info baked into RA!

**Comment by WITMiddle** [comment]
Removing this section and associated PR. I will integrate the content into other sections.

**Pull request closed without merging**